### ISSUE TITLE
Only plays '.aiff' sounds

### ISSUE THREAD

**Issue by likethemammal** [adriancooney/node-sfx#3]
I wasn't able to get this to play `.mp3`s. After looking through the source, I realized you only used `.aiff` sounds. Luckily I was able to convert my files.

Can you add a section in the readme describing the supported audio formats? Or is it somehow based on the OS specifically? I'm using Linux at the moment.

**Comment by adriancooney** [comment]
Yeah, it looks like a Linux issue. If you look inside the [`/platforms` folder](https://github.com/adriancooney/node-sfx/tree/master/platform), you can check out how `node-sfx` actually plays the sound for each platform. It simply runs a command that plays audio with a path to the sound file. It looks like `paplay`, the command the Linux platform uses, doesn't support MP3. We could change the command to something that does support MP3 but we'd probably have to introduce a new dependency. Thoughts?

**Comment by likethemammal** [comment]
My main reason for choosing this package, as opposed to others on npm that play sounds, was because it didn't require any external install of a CLI player.

As long as that dependancy isn't outside npm I'm golden. But seeing as Audacity had no problem exporting to aiff, it's not really an issue for me anymore.
